app/Http/Controllers/PurchaseOrderController.php

```
@@ -40,4 +54,25 @@ public function store(PurchaseOrderRequest $request)
+    public function Update(PurchaseOrderRequestFilter $request, PurchaseOrderRequest $purchaseOrderRequest, $id)
+    {
+        $validatedData = $purchaseOrderRequest->validated();
+        $purchaseOrder = PaymentRequest::purchaseOrder()->findOrFail($id);
+        $purchaseOrder->update($validatedData);
+        $purchaseOrder->details()->delete();
+        $purchaseOrder->details()->createMany($validatedData['details']);
+        return (new PaymentRequestCollection($purchaseOrder))->additional([
+            'success' => true,
+            'message' => 'Purchase Order Successfully Updated.',
+        ]);
+    }
```

PHPMD (2.15.0)

The method Update is not named in camelCase. (undefined)

(CamelCaseMethodName)